alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -47,6 +47,14 @@ impl Pty {
     ) -> Self {
         Self { backend: backend.into(), conout: conout.into(), conin: conin.into(), child_watcher }
     }
+
+    pub fn fd(&self) -> isize {
+        self.child_watcher.fd()
+    }
```

Comment:
You should also document what exactly is getting returned here, because `isize` is clearly confusing. Maybe add a new type for that.

Also, can you generally close the alacritty terminal that way? Though, I think if you close it in between it won't create issues other than telling that pty got closed?

Reply:
Since its the process handle, people could do anything to it. Maybe make this function `unsafe`?
As this is just my initial idea, I wanted to get this information directly, so I simply wrote codes to get them. Given that this PR is still very immature, I have marked it as a draft. Any suggestions are welcome

Reply:
I think keeping it safe is fine given that functions to manipulate with that `HANDLE` will be unsafe in rust, so it's kind of the same as with pointer, because getting it is safe, but touching is unsafe.